sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -308,6 +308,99 @@ export function getCountDetailsOrUndefined(value: any): MessageCountDetails | un
   };
 }
 
+/**
+ * Service expects the time duration in ISO-8601 time duration format.
+ * Example: PT10M, P106DT2H48M5.47S
+ *
+ * This helper method is to convert the duration in seconds to ISO-8601 duration format.
+ *
+ * https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm
+ *
+ * @export
+ * @param {(number | undefined)} timeInSeconds
+ * @returns {(string | undefined)}
+ */
+export function getISO8601DurationFromSeconds(
+  timeInSeconds: number | undefined
+): string | undefined {
+  if (timeInSeconds == null) {
```

Comment:
can we make this more explicit? 

```suggestion
  if (timeInSeconds === undefined || timeInSeconds === null) {
```

Reply:
This is interesting - why would we want to expand it? I was under the impression this was good practice?

https://basarat.gitbook.io/typescript/recap/null-undefined

Reply:
Many developers are not familiar with the subtleties of non-strict comparisons, so they are often banned by lint rules. It's also easy to not notice a missing extra =.

Reply:
And if you want to take it from Anders, we shouldn't ever use `null` in our code since it's not required by the language, but I'm not that much of a purist as to avoid it for use in sentinel values. ;)

Reply:
Interesting. 
`timeInSeconds === undefined || timeInSeconds === null`

To handle both null and undefined, we have been doing `timeInSeconds == undefined` check at other places.